samples/NativeAOT/NativeAotTypeManager.cs

```diff
@@ -0,0 +1,206 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using Java.Interop;
+using Java.Interop.Tools.TypeNameMappings;
+
+namespace NativeAOT;
+
+partial class NativeAotTypeManager : JniRuntime.JniTypeManager {
+
+	internal const DynamicallyAccessedMemberTypes Methods = DynamicallyAccessedMemberTypes.PublicMethods | DynamicallyAccessedMemberTypes.NonPublicMethods;
+	internal const DynamicallyAccessedMemberTypes MethodsAndPrivateNested = Methods | DynamicallyAccessedMemberTypes.NonPublicNestedTypes;
+
+	// TODO: list of types specific to this application
+	Dictionary<string, Type> typeMappings = new () {
+		["android/app/Activity"]                = typeof (Android.App.Activity),
+		["android/content/Context"]             = typeof (Android.Content.Context),
+		["android/content/ContextWrapper"]      = typeof (Android.Content.ContextWrapper),
+		["android/os/BaseBundle"]               = typeof (Android.OS.BaseBundle),
+		["android/os/Bundle"]                   = typeof (Android.OS.Bundle),
+		["android/view/ContextThemeWrapper"]    = typeof (Android.Views.ContextThemeWrapper),
+		["my/MainActivity"]                     = typeof (MainActivity),
+	};
+
+	public NativeAotTypeManager ()
+	{
+		AndroidLog.Print (AndroidLogLevel.Info, "NativeAotTypeManager", $"# jonp: NativeAotTypeManager()");
+
+		// TODO: should eventually be refactored in Mono.Android.dll
+		var jniToManaged = TypeManagerMapDictionaries.JniToManaged;
```

Comment:
How/why are these needed? They're used by `TypeManager.CreateInstance()`, but as that's currently full of P/Invokes, we're apparently not hitting `TypeManager.CreateInstance()`, so why do we need to access & update these dictionaries?